srcpkgs/caffeine-ng/template

```diff
@@ -1,26 +1,19 @@
 # Template file for 'caffeine-ng'
 pkgname=caffeine-ng
-version=3.5.1
-revision=2
+version=4.0.2
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm"
 depends="desktop-file-utils gtk+3 hicolor-icon-theme libnotify
```

Comment:
changelog lists a bunch of deps no longer needed

Reply:
The version at the top of the changelog doesn't have a [release](https://codeberg.org/WhyNotHugo/caffeine-ng/tags) yet.

Reply:
aha, i misread